package.json

```diff
@@ -17,12 +17,11 @@
     "check:prettier": "prettier --check .",
     "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
     "build": "tsc --build",
-    "fetch-schemas": "pnpm --filter auth --filter backend fetch-schemas",
     "localenv": "docker compose -f ./infrastructure/local/docker-compose.yml -f ./infrastructure/local/peer-docker-compose.yml",
     "localenv:build": "docker compose -f ./infrastructure/local/docker-compose.yml -f ./infrastructure/local/peer-docker-compose.yml -f ./infrastructure/local/build-override.yml build",
     "localenv:seed:auth": "pnpm -C ./packages/auth knex seed:run --env=development && pnpm -C ./packages/auth knex seed:run --env=peerdevelopment",
     "localenv:dbvolumes:remove": "docker volume rm local_database-data && docker volume rm local_tigerbeetle-data",
-    "sanity": "pnpm fetch-schemas pnpm -r build && pnpm -r test"
```

Comment:
What was the `sanity` for, originally? Should anything else belong here?

Reply:
I don't know why it was added
- https://github.com/interledger/rafiki/pull/458